apps/expo/src/app/muscleGroup/index.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useState } from 'react'
+import { Dimensions, Image, Text, View, StyleSheet } from 'react-native'
+import { router } from 'expo-router'
+import tailwind from '@/tooling/tailwind'
+import { Ionicons, MaterialCommunityIcons } from '@expo/vector-icons'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import Body from 'react-native-body-highlighter'
+import ToggleSwitch from '~/components/toggle/Toggle'
+import FrontBackSwitch from '~/components/muscleGroup/FrontBackSwitch'
+
+export type muscleSelect =
+  | "abs"
+  | "adductors"
+  | "ankles"
+  | "biceps"
+  | "calves"
+  | "chest"
+  | "deltoids"
+  | "deltoids"
+  | "feet"
+  | "forearm"
+  | "gluteal"
+  | "hamstring"
+  | "hands"
+  | "hair"
+  | "head"
+  | "knees"
+  | "lower-back"
+  | "neck"
+  | "obliques"
+  | "quadriceps"
+  | "tibialis"
+  | "trapezius"
+  | "triceps"
+  | "upper-back";
+
+export default function NewWorkoutMuscleGroup() {
+  const { width: screenWidth, height: screenHeight } = Dimensions.get('window')
+  const vmin70 = Math.min(screenWidth, screenHeight) * 0.7
+
+  const [bodyPartSelected, setBodyPartSelected] = useState({/*
+    slug: "biceps",
+    intensity: 2,*/
+  }
+  )
+  const [isBackSideEnabled, setIsBackSideEnabled] = useState(false)
+  const [isMale, setIsMale] = useState(true)
+  const toggleSwitch = () =>
+    setIsBackSideEnabled((previousState) => !previousState)
+
+  const toggleGenderSwitch = () => setIsMale((previousState) => !previousState)
+
+  //let tmp = "";
+  let sameGroup = {};
+  const [tmp1, setTmp1] = useState({})
+  const [userSelection, setUserSelection] = useState("");
+
+  return (
+    <SafeAreaView style={{ backgroundColor: '#1C1B1B', flex: 1 }}>
+      <View className="flex flex-row justify-between px-5">
+        <Ionicons
+          onPress={() => router.back()}
+          name="chevron-back"
+          size={24}
+          color="#CACACA"
+        />
+        <Text style={{ color: '#CACACA', fontSize: 20 }}>Select a muscle group</Text>
+        <Text></Text>
+      </View>
+
+      <View className="flex-1"
+            style = {{alignItems: "center", justifyContent: "center"}}>
+
+        <Body
+
+          onBodyPartPress={(e) =>
+            {
+              switch (e.slug) {
+                case "deltoids":
+                  // Shoulder
+                  setBodyPartSelected({ slug: e.slug, intensity: 2 })
```

Comment:
handleMusclePress() or smth

Reply:
I'll add additional selection functionality to <Body> later